disco_aws_automation/disco_group.py

```diff
@@ -67,7 +67,7 @@ def list_groups(self):
 
     def get_instances(self, hostclass=None, group_name=None):
         asg_instances = self.autoscale.get_instances(hostclass=hostclass, group_name=group_name)
-        asg_instances = [instance.__dict__ for instance in asg_instances]
+        asg_instances = [instance for instance in asg_instances]
```

Comment:
I believe this is not needed anymore. `self.autoscale.get_instances(hostclass=hostclass, group_name=group_name)` returns  list.